src/sshagent/OpenSSHKey.cpp

```diff
@@ -372,10 +372,21 @@ bool OpenSSHKey::openPrivateKey(const QString& passphrase)
             return false;
         }
 
-        QCryptographicHash hash(QCryptographicHash::Md5);
-        hash.addData(passphrase.toUtf8());
-        hash.addData(m_cipherIV.data(), 8);
-        QByteArray keyData = hash.result();
+        QByteArray keyData;
+        QByteArray mdBuf;
+        mdBuf.resize(0);
```

Comment:
Freshly constructed QByteArray is zero length so this would be superfluous.

Reply:
True enough. I'll drop that.